aspnetcore/mvc/views/working-with-forms.md

```diff
@@ -584,16 +584,16 @@ Supported [AnchorTagHelper](xref:mvc/views/tag-helpers/builtin-th/anchor-tag-hel
 
 ### Submit to controller example
 
-The following example submits the form to the `Index` action of `HomeController` if either the input or button were clicked:
+The following code submits the form to the `Index` action of `HomeController` when the input or button are selected:
 
 ```cshtml
 <form method="post">
     <button asp-controller="Home" asp-action="Index">Click Me</button>
-    <input type="image" src="..." alt="Or Click Me" asp-controller="Home" asp-action="Index" />
+    <input type="image" src="..." alt="Alt" asp-controller="Home" asp-action="Index" />
```

Comment:
Leave as `Or Click Me`. The `<input type="image">` typically relies on its src to say the click me text but obv in this format we don't have that luxury.

Reply:
I took that out to make to remove the horizontal scroll bar. I'll put that back and make it two lines.